config/config.go

```diff
@@ -837,6 +837,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.audiencenetwork.disabled", true)
 	v.SetDefault("adapters.audiencenetwork.endpoint", "https://an.facebook.com/placementbid.ortb")
 	v.SetDefault("adapters.avocet.disabled", true)
+	v.SetDefault("adapters.axonix.disabled", true)
```

Comment:
I am curious if there is a specific reason to default disabled. Is it just to have more control over who is using your adapter?

Reply:
Yeah, we prefer to enable our supply partners on a one by one basis.